### ISSUE TITLE
Raw data should be put in a reserved folder.

### ISSUE THREAD

**Issue by cpicanco** [pupil-labs/pupil#241]
Hi,

The current folder tree for Pupil Capture outputs is as follows:

```
- recordings
  - yyyy-mm-dd
    - idx
```

I am suggesting to put Pupil Player outputs in a separated folder. Either:

```
- recordings
  - yyyy-mm-dd
    - idx
      - analytics
```
or

```
# mirrored from the recordings file tree
- analytics
  - yyyy-mm-dd
    - idx
```

Is good practice to keep the raw data as read only. Putting in a different folder makes it easier to do so and also decrease the chances of getting lost in a bunch of files (as I am right now :smile: )

Other possible names: `player_outputs`, `player_exports`, `players_analytics`...

**Comment by mkassner** [comment]
This is a nice idea. I ll vote for:

```
- recordings
  - yyyy-mm-dd
    - idx
      - analytics
```

**Comment by willpatera** [comment]
I agree with @mkassner -- let's add a milestone to this issue.

**Comment by willpatera** [comment]
Fixed in 3629b145ca23073cb716c8d9a25ede14d6c9a030
